### DISCUSSION TITLE
Getting error "Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Failed to start bean 'org.springframework.kafka.config.internalKafkaListenerEndpointRegistry'; nested exception is org.apache.kafka.common.errors.TimeoutException: Timeout expired while fetching topic metadata"

### DISCUSSION THREAD

**Discussion by Himanshu-Gupta-maersk** [spring-projects/spring-kafka#3229] [General]
I am getting below error while running spring boot application executable jar on linux machine. I am trying to connect remote topic from LAN server. I had asked network team to open all firewalls. I had tried many way to fix but getting same errors. please suggest the possible cause and solution of this.

 Exception encountered during context initialization - cancelling refresh attempt: org.springframework.context.ApplicationContextException: Failed to start bean 'org.springframework.kafka.config.internalKafkaListenerEndpointRegistry'; nested exception is org.apache.kafka.common.errors.TimeoutException: Timeout expired while fetching topic metadata

**Comment by artembilan** [comment]
Can you share the whole stack trace, please, and as a code snippet?
Right now your question is just not readable.

Thanks
